tests/compiletest.rs

```diff
@@ -50,33 +52,35 @@ fn run_tests(mode: Mode, path: &str, target: &str, with_dependencies: bool) -> R
         stdout_filters: STDOUT.clone(),
         root_dir: PathBuf::from(path),
         mode,
-        program: miri_path(),
+        program: CommandBuilder::rustc(),
```

Comment:
Might be worth a comment that this is fixed up to use miri later?

Does this indicate a missing API for constructing a rustc-like CommandBuilder with a custom path?

Reply:
Well I guess we are fixing up the args as well so :shrug: